tests/provision/bootc/data/plans.fmf

```diff
@@ -15,15 +15,15 @@ execute:
             add-tmt-dependencies: true
             container-image: localhost/tmt-bootc-needs-deps
         environment:
-            PATTERN: localhost/tmtmodified
+            PATTERN: localhost/tmt-deps
 
     /includes-deps:
         summary: "Image that already includes dependencies"
         provision+:
             add-tmt-dependencies: false
             container-image: localhost/tmt-bootc-includes-deps
         environment:
-            PATTERN: localhost/tmt-bootc-includes-deps
+            PATTERN: localhost/tmt-deps
```

Comment:
If the image includes all necessary dependencies, shouldn't it be used directly? This suggests that we create a new image even if it is not necessary. I guess there could be situations when user really wants to boot their prepared image without any changes performed by `tmt`? @thrix, any thoughts on this?